test/journeys/specs/space/dataApi/startup-settings.js

```diff
@@ -0,0 +1,226 @@
+import '@ciscospark/plugin-logger';
+import CiscoSpark from '@ciscospark/spark-core';
+import testUsers from '@ciscospark/test-helper-test-users';
+import '@ciscospark/internal-plugin-conversation';
+
+import {moveMouse} from '../../../lib/test-helpers';
+import {elements} from '../../../lib/test-helpers/space-widget/main';
+import {elements as messageElements} from '../../../lib/test-helpers/space-widget/messaging';
+import {answer, hangup, elements as meetElements} from '../../../lib/test-helpers/space-widget/meet';
+import {constructHydraId} from '../../../lib/hydra';
+
+describe('Widget Space', () => {
+  describe('Data API Settings', () => {
+    const browserLocal = browser.select('1');
+    const browserRemote = browser.select('2');
+    let docbrown, lorraine, marty;
+    let conversation;
+
+    before('load browsers', () => {
+      browser
+        .url('/data-api/space.html')
+        .execute(() => {
+          localStorage.clear();
+        });
+    });
+
+    before('create marty', () => testUsers.create({count: 1, config: {displayName: 'Marty McFly'}})
```

Comment:
We could use your new helper functions here, correct?

Reply:
yea, just a spot i didn't touch because it wasn't erroring out regularly.

Reply:
you forgot "yet", lol